scripts/e2e-widget.js

```diff
@@ -18,7 +18,8 @@ const tabularFileNames = [
   "responsibleaidashboard-housing-classification-model-debugging",
   "responsibleaidashboard-diabetes-decision-making",
   "responsibleaidashboard-housing-decision-making",
-  "responsibleaidashboard-multiclass-dnn-model-debugging"
+  "responsibleaidashboard-multiclass-dnn-model-debugging",
+  "responsibleaidashboard-orange-juice-forecasting"

```

Comment:
should this be put in a separate forecasting section instead of under tabular, vision or text?

Reply:
Forecasting is tabular. Do you mean because the dashboard looks different? The tests are able to distinguish based on the test configuration, of course. I have put it in a separate nb_group in the e2e workflow yaml, but only because the data balance flight won't do anything different for it...